raiden/network/utils.py

```diff
@@ -99,10 +99,7 @@ def get_http_rtt(
             durations.append(
                 requests.request(method, url, timeout=timeout).elapsed.total_seconds()
             )
-        except (RequestException, OSError):
-            return None
-        except Exception as ex:
-            print(ex)
+        except Exception:
```

Comment:
Humm, this doesn't look right, instead of catching `Exception` we should remove that handler and expand the one for the network related errors.

Reply:
It looks awkward, yes. Apparently there is a lot that can go wrong here (including `MatrixRequestError`s or different connection problems that are not caught by `OSError`) and we don't want to risk a crash because of this. Related issue: https://github.com/raiden-network/raiden/issues/1793

If you aren't happy with the catch-all `except`, we could silently swallow the known possible exceptions and log others as errors instead.

Reply:
> It looks awkward, yes. Apparently there is a lot that can go wrong here (including MatrixRequestErrors or different connection problems that are not caught by OSError) and we don't want to risk a crash because of this

I don't think this piece of code can raise `MatrixRequestErrors`. The `try` blocks is only wrapping the `requests` library. The stacktraces from the linked issue are unrelated.

> If you aren't happy with the catch-all except, we could silently swallow the known possible exceptions and log others as errors instead.

Please, do it only for the necessary requests exceptions

Reply:
@ulope @andrevmatos can any of you two confirm that a matrix error is raised here and how that would happen?

Reply:
That was my suggestion to Jochen to add the `MatrixError`, but I apparently didn't look close enough. This is only using `requests`, so there is no way it can raise a `MatrixError`.

The way I remember it the problem was that we had cases where servers were in weird states that caused errors here that weren't being caught by only `(RequestsError, OSError)`.

The line was added as part of the fix to #1793.

However since MatrixError makes no sense I'd say er remove it and see if any errors pop up again. If so we can add the proper exception class and add a comment on why it's there.